php/classes/controllers/class-players-controller.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace SeriouslySimplePodcasting\Controllers;
+
+use SeriouslySimplePodcasting\Renderers\Renderer;
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Players_Controller class
+ *
+ * @author      Danilo Radovic
+ * @category    Class
+ * @package     SeriouslySimplePodcasting/Controllers
+ * @since       2.3
+ */
+class Players_Controller extends Controller {
+
+	public $renderer = null;
+	public $episode_controller;
+
+	public function __construct( $file, $version ) {
+		parent::__construct( $file, $version );
+		$this->renderer           = new Renderer();
+		$this->episode_controller = new Episode_Controller( $file, $version );
+		$this->init();
+	}
+
+	public function init() {
```

Comment:
@daniloradovic during writing the inital Players_Controller_Test for this, I realised the scripts and styles where not being triggered on the wp_enqueue_scripts hook. I've updated the class to do this. Otherwise the scripts and styles will be loaded whenever the class is instantiated, not only when they need to be for the front end rendering.

I also moved the shortcode registration hook to this init method in the class.